src/js/lib/navigate.js

```diff
@@ -42,3 +42,17 @@ export function jumpTo (from, to) {
     stacks(from).immediatelyResetStack([Router.getRoute(to)], 0);
   });
 }
+
+/**
+ * popToTop redirects to the top of the stack
+ * @param {object} nav - navigtion prop
```

Comment:
We should make this function consistent with the others.  The others functions don't get the navigation object as an argument, but instead from `NavigationActions`.